fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/indexes/MultiDimensionalIndexHelper.java

```diff
@@ -63,6 +63,10 @@ public static RTree.Config getConfig(@Nonnull final Index index) {
         if (rtreeStorage != null) {
             builder.setStoreHilbertValues(Boolean.parseBoolean(rtreeStoreHilbertValues));
         }
+        final String rtreeUseNodeSlotIndex = index.getOption(IndexOptions.RTREE_USE_NODE_SLOT_INDEX);
+        if (rtreeUseNodeSlotIndex != null) {
+            builder.setUseNodeSlotIndex(Boolean.parseBoolean(rtreeUseNodeSlotIndex));
+        }
```

Comment:
I believe that this (in connection with the fact that the default value for `useNodeSlotIndex` in the builder is `true`) means that multi-dimensional indexes will use the node slot index by default. In principle that sounds fine, though it does mean that when someone upgrades to this version of the Record Layer, if they have any pre-existing indexes, they will need to remember to add the index option or their index will be corrupted. If we don't care about data integrity (at this stage of development), then that's fine; otherwise, I think we need to default this to `false`.

Reply:
There is currently no one using this so I am fine breaking that.